.github/workflows/test-os.yml

```diff
@@ -110,3 +110,10 @@ jobs:
         run: |
           vagrant ssh -- "cp /vagrant/hack/dockerfiles/freebsd-smoke.Dockerfile /vagrant/cmd/buildctl/Dockerfile"
           vagrant ssh -- "cd /vagrant/cmd/buildctl; go run . build --frontend dockerfile.v0 --local context=. --local dockerfile=."
+
+      - name: Archive integration test artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: freebsd-run-logs
+          path: run-logs
+          retention-days: 5
```

Comment:
We should always run this step, I think this only runs if the previous job succeeds.

Also, if we could print out the logs similar to how we do for the other tests, see https://github.com/moby/buildkit/actions/runs/5737822387/job/15550263885?pr=4089#step:19:3.

Reply:
Thanks, fixed. I also changed the test image tag, for the old one was removed.

Reply:
Ah sorry @jedevc, I missed this `upload-artifact` has been removed. Looks good then :+1: